web/src/main/java/com/futurewei/alcor/web/entity/dataplane/GoalStateJsonNB.java

```diff
@@ -0,0 +1,94 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.constants.Common;
+import com.futurewei.alcor.common.entity.CustomerResource;
+import com.futurewei.alcor.web.entity.securitygroup.SecurityGroupEntity;
+import com.futurewei.alcor.web.entity.subnet.SubnetEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+
+public class GoalStateJsonNB extends CustomerResource {
```

Comment:
GoalState implies to that this is a "full" state, which might or might not be. Let us use a more generic name like "NetworkConfiguration".

Reply:
Let us not inherit from CustomerResource as not all fields are customer fields.